### DISCUSSION TITLE
[WARNING] [OglModel(OglModel)] Could not read value for data field translation: np.float64(0.0) np.float64(0.0) np.float64(0.0)

### DISCUSSION THREAD

**Discussion by ozilxu** [SofaDefrost/SoftRobots#298] [Help - Using] [answered]
When I ran the code from https://github.com/SofaDefrost/SoftRobots/blob/master/examples/tutorials/CableGripper/details/step4.py. There is nothing shown in the scene, and the terminal shows that it cannot load the data through Numpy. 
![1](https://github.com/user-attachments/assets/59d78b5c-a5e0-4b49-8cd9-e182097473d3)

**Comment by ozilxu** [comment, marked as answer]
Hello, guys, the problem is solved by reinstalling the numpy version from 2.1.0 to 1.26.4.

**Reply by EulalieCoevoet** [reply]
Thank you for reporting this!
